### DISCUSSION TITLE
FR - Hide Commands

### DISCUSSION THREAD

**Discussion by hdbordercollie** [LostPaul/obsidian-folder-notes#226] [Ideas]
Hiya. I've never used the canvas folder note option on the file explorer right click menu, nor have I used the exclude folder from folder note option, nor have I used the longwinded commands that just clutter up my commad palette. It'd be great if I could have a toggle for all of these, as using the plugin in a minimalist vault is a pain.

**Comment by lukiwieser** [comment]
Hi, I would also appreciate the option to hide commands from the right click menu.

**Comment by LostPaul** [comment]
For the context menu you could use https://github.com/mara-li/obsidian-context-menu-hider and for the command palette https://github.com/RyotaUshio/obsidian-command-block-list
